packages/web-components/src/components/button-group/__stories__/README.stories.mdx

```diff
@@ -49,6 +49,13 @@ import 'carbon-web-components/es/icons/arrow--right/20.js';
 </dds-button-group>
 ```
 
+### @click example
+
+```html
+<dds-button-group>
+  <dds-button-group-item @click=${this._handleClick}></dds-button-group-item>
```

Comment:
Might want others' opinions as well, but wondering if the example method should be something like `${myCustomClickHandler}` rather than `${this_handleClick}`?